packages/vue/src/components/library/card-notes.vue

```diff
@@ -44,7 +44,10 @@
               >
                 <div class="note-title">
                   <div>{{ formatDate(new Date(note.date)) }} - {{ note.manager }}</div>
-                  <dx-button icon="overflow" />
+                  <dx-button
+                    icon="overflow"
+                    stylingMode="text"
```

Comment:
github actions says that for vue files Attribute 'stylingMode' must be hyphenated

Reply:
Fixed